nupic/frameworks/opf/clamodel.py

```diff
@@ -1486,4 +1482,79 @@ def getNextRecordDict(self):
 
 
 
-
+########################
+# example model confuguration: used in tests, examples, etc
+# use: from nupic.frameworks.opf.clamodel import modelConfig
+modelConfig = (
```

Comment:
I really don't like having this config in the production class. I liked it much better in the test. Please move it back?